docs/lua-records/reference/comboaddress.rst

```diff
@@ -0,0 +1,112 @@
+.. _ComboAddress:
```

Comment:
Wonder if we could symlink this to the dnsdist comboaddress docs?

Reply:
The content is not exactly the same as I also documented the `ComboAddressSet` functionalities.